tests/conftest.py

```diff
@@ -12,6 +13,18 @@
 from cachito.web.config import TEST_DB_FILE
 from cachito.web.app import create_app
 
+try:
+    from tests.temp_env_var import TEMP_ENV_VARS, ENV_VARS_TO_SUSPEND
+except ImportError:
+    TEMP_ENV_VARS = {}
+    ENV_VARS_TO_SUSPEND = []
```

Comment:
Why this?

Reply:
If they are missing from the temp_env_vars.py file, it will define them as empty to avoid undefined key errors later.